Oracles/chainlink-prices-usd/db.schema.ts

```diff
@@ -0,0 +1,21 @@
+import { String, Array } from "@blockflow-labs/utils";
+
+//DB1 will include 3-4 params as discuused related to price value 
+export interface PriceDB{
+  id: String;
+  name: String;
+  symbol: String;
+  decimals: Number;
+  quote_currency: String;
+}
+
+//DB2 is chainlink pair and has other info
+//should i store all these big numbers where big number js gonna be used as numbers or string
+export interface chainlink_pair{
+    id: String;
+    update_count: Number;
+    transanction_hash: String;
+    last_block_number: Number;
+    round_id: Number;
+    impl_update: Number;
```

Comment:
Use camelCase. and remove impl_update

Reply:
Sir this was just a copy of studio.schema.ts to write comments, i have removed it as of now.